terminus-ui/input/src/input.component.md

```diff
@@ -320,6 +320,7 @@ Supported event emitters:
   (cleared)="inputWasBlurred($event)"
   (inputFocus)="inputGainedFocus($event)"
   (selected)="newDateSelected($event)"
+  (inputPaste)="inputValuePasted($event)"
```

Comment:
Just below this our usage docs state `Each emitter is called with the current input value.` - can you just remove this line for now? (I'll go in the future and update the docs format to show the payload for each output to match our newer docs format)